webapp/src/components/ResourceList.tsx

```diff
@@ -281,27 +285,14 @@ type RDCModalProps = {
 const ResourceDeleteConfirmationModal: React.FC<RDCModalProps> = (props: RDCModalProps) => {
   return (
     <Modal close={props.close} title="Confirm Deletion" submit={props.submit} isValid={true} submitText="Delete" isWarning={true}>
-      <p>Are you sure you want to delete the following resources?</p>
-      {props.resources.map(r => <p>{r.title}</p>)}
+      <p className="delete-header">Are you sure you want to delete the following resources?</p>
```

Comment:
What if the CSS class for this message was consistent across our modals? Maybe like `message` or something? That way we can style all of them at the same time. (If so, also change the `ArchiveResourceConfirmationModal` mesage to use that class.)